bot/pagination.py

```diff
@@ -37,12 +37,19 @@ class LinePaginator(Paginator):
         The suffix appended at the end of every page. e.g. three backticks.
     * max_size: `int`
         The maximum amount of codepoints allowed in a page.
+    * scale_to_size: `int`
+        The maximum amount of characters a single line can scale up to.
     * max_lines: `int`
         The maximum amount of lines allowed in a page.
     """
 
     def __init__(
-        self, prefix: str = '```', suffix: str = '```', max_size: int = 2000, max_lines: int = None
+        self,
+        prefix: str = '```',
+        suffix: str = '```',
```

Comment:
I personally have no idea, but I can try to take a look at the current places where LinePaginator is used to see if there are any decent use cases for the prefix and suffix parameters. From looking at the git blame, it looks like it was all added at once a couple of years ago: https://github.com/python-discord/bot/pull/25. It could very well be a common case of feature bloat, but I'm not confident enough about the areas in which `LinePaginator` is used by the bot to know if those params are useful.

Reply:
well, if it was added a couple of years ago, I suggest we could probably just remove these as arguments and make them constants if (and only if) there are zero usages in the codebase. If we've never used those arguments in 2 years, we're not gonna start using them tomorrow.

Would you mind taking a quick look to see if you can find any usages?

Reply:
@lemonsaurus

From looking through all of the results in `git grep "LinePaginator"`, I found just one actual usage for *prefix* in the bot repo:

https://github.com/python-discord/bot/blob/232cc68b0a0ef210027beacb9b4f50ffeeaddd00/bot/cogs/help.py#L296-L302

Since this would require some additional investigation to see if the same functionality could be easily implemented without it, I think we should probably consider removal of *prefix* and *suffix* in a separate issue and PR (if at all, since you mentioned that they should only be removed IFF there are zero usages). Arguably, we could just make suffix a constant, but to me it seems like it would be a bit odd to just remove one without the other.

Would this be acceptable? While I think it would be useful to remove these parameters if they're not really being used, I'm definitely leaning towards handling it independently from this PR.

Reply:
I agree that it is better left for another PR, though I don't have a problem with it just being left as-is.

Reply:
if we're using it we can just keep it.

Reply:
Alright, sounds good to me.